artman/config/loader.py

```diff
@@ -54,7 +54,7 @@
 }
 
 
-def load_artifact_config(artman_config_path, artifact_name):
+def load_artifact_config(artman_config_path, artifact_name, aspect=None):
```

Comment:
Will  `aspect='ALL'` be appropriate here? I hope it can allow to not specify default value for aspect parameters in `test_code_generation.py` file (multiple times).

Reply:
Oh, I could actually remove the aspect parameters from `test_code_generation.py` as it is; `aspect=None` has the same result as `aspect='ALL'`.

Reply:
So it turns out that `test_code_generation.py` needs the parameter to be set, because `test_code_generation.py` doesn't call `load_artifact_config` directly.